<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QgsGrassModuleBase</class>
 <widget class="QWidget" name="QgsGrassModuleBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>GRASS Module</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>10</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="mTabWidget" >
     <widget class="QWidget" name="options" >
      <attribute name="title" >
       <string>Options</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="output" >
      <attribute name="title" >
       <string>Output</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="Q3TextBrowser" name="mOutputTextBrowser" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Manual</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="Q3TextBrowser" name="mManualTextBrowser" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Q3ProgressBar" name="mProgressBar" />
   </item>
   <item>
    <widget class="Q3Frame" name="frame4" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>10</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="mRunButton" >
        <property name="text" >
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>135</width>
          <height>26</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="mCloseButton" >
        <property name="text" >
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>26</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3ProgressBar</class>
   <extends>QWidget</extends>
   <header>q3progressbar.h</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends></extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>Q3TextBrowser</class>
   <extends>QWidget</extends>
   <header>q3textbrowser.h</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
